pkg/names/names.go

```diff
@@ -60,6 +60,11 @@ func GenStsName(vdb *vapi.VerticaDB, sc *vapi.Subcluster) types.NamespacedName {
 	return GenNamespacedName(vdb, vdb.Name+"-"+sc.GenCompatibleFQDN())
 }
 
+// GenSandboxConfigMapName returns the name of the sandbox config map
+func GenSandboxConfigMapName(vdb *vapi.VerticaDB, sandbox string) types.NamespacedName {
```

Comment:
Roy added a similar function in his PR (#785) called `GenConfigMapName`, although I prefer your name here because its clear that the configMap is for the sandbox. We should make sure we don't end up with two functions that do the same thing.

Reply:
I will wait for Roy's PR(#785) to be merged first, then I can update his function.